app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Username</h1>
\ No newline at end of file
+<ul>
+  <% @users.each do |user| %>
+    <li class="card">
+      <div class="photo">
+        <%= image_tag(user.photo, width: "150px") %>
+      </div>
+      <div class="user_card">
+        <div class="user_name"><%= user.name %></div>
+        <div class="user_posts">Number of posts: <%= user.posts_counter %></div>
+      </div>
```

Comment:
- [ ] Following the project requirements, clicking on each user should take me to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). Please add a click event that.